README.md

```diff
@@ -56,5 +56,19 @@ Simple queueing service built for Internet Computer
 - Send & read messages simultaneously
 - Once processed only
 
+---------
 
+#### <b>DFX Commands for local testing</b>
 
+```
+$ dfx deploy --mode=reinstall --argument '(opt record {principal "woicf-w4c3l-ggagd-ktql2-zzwf6-xwrx7-ehupq-an62o-uvf5d-owgvw-nae"; null})' icsqs  
```

Comment:
I was expecting the principlay to be read from 
$(dfx indetnity get-principal) and use it inside the sting